### ISSUE TITLE
No field for AccessibilityManager

### ISSUE THREAD

**Issue by LukeAvery92** [pixijs/pixi-haxe#100]
Pixi JS documentation has a field for object.accessibility = true/false. We no example of how to use this haxe externs AccessibilityManager it doesn't seem to work as expected.

My setup, Create renderer, create AccessibilityManager, set objects interactive to true and add some mouse events.

I can see mouse events working but tabbing is not.

**Comment by adireddy** [comment]
I haven't used `AccessibilityManager` before but noticed that the properties like `accessible`, `accessibleTitle`, `tabIndex`, etc are missing.

**Comment by adireddy** [comment]
Available in v4.2.4, let me know if it works or not.
